app/src/main/java/mozilla/lockbox/view/AppWebPageFragment.kt

```diff
@@ -62,4 +64,15 @@ class AppWebPageFragment : BackableFragment(), WebPageView {
         }
         webView.loadUrl(url)
     }
-}
+
+    override fun handleNetworkError(networkErrorVisibility: Boolean) {
+        if (!networkErrorVisibility) {
+            errorHelper.showWebPageNetworkError(view!!)
+        } else {
+            errorHelper.hideWebPageNetworkError(view!!)
+        }
+    }
+
+    override val retryNetworkConnectionClicks: Observable<Unit>
+        get() = view!!.networkWarning.retryButton.clicks()
```

Comment:
What does this button do?

As I understand it, it will hide the error message snack bar, but not do anything related to what we were going to use the network for.